src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -76,10 +77,12 @@
          :keyboardVerticalOffset (- (max 20 (:bottom insets)))}
         [page-nav]
         [pin.banner/banner chat-id]
-        [messages.list/messages-list chat]
+        [messages.list/messages-list chat insets]
         (if-not able-to-send-message?
           [contact-requests.bottom-drawer/view chat-id contact-request-state group-chat]
-          [composer/composer chat-id insets])])]))
+          ;[composer/composer chat-id insets]
+          [bottom-sheet-composer/bottom-sheet-composer insets]
```

Comment:
I am still keeping the old composer (until the tasks mentioned above is complete). Will switch back to the old composer after QA review, before merging.